src/os/memstore/MemStore.cc

```diff
@@ -566,35 +566,35 @@ class MemStore::OmapIteratorImpl : public ObjectMap::ObjectMapIteratorImpl {
     : c(c), o(o), it(o->omap.begin()) {}
 
   int seek_to_first() override {
-    std::lock_guard<std::mutex>(o->omap_mutex);
+    std::lock_guard<std::mutex> lock(o->omap_mutex);
```

Comment:
nit, no need to specify the `std::mutex` template argument anymore, as we've switched to C++17.